docs/multisig-transactions.md

```diff
@@ -0,0 +1,177 @@
+
+### Create the address
+Now you will want to create the payment address:
+```sh
+cardano-cli address build \
+--payment-script-file payment.script \
+--stake-script-file stake.script \
+--out-file payment.addr \
+--mainnet
+```
+Or you can swap `--mainnet` for `--testnet-magic 1097911063` if you want to create a testnet address.
+
+## Creating the transaction
+Now we have created a multisig address and we are able to continue with creating a simple transaction. We will assume you have already send some funds to your newly created address (for testnet addresses you can use the [Faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/)).
+
+### Querying the UTxO
+First we need to query for the UTxO:
+```sh
+cardano-cli query utxo \
+--address $(cat payment.addr) \
+--mainnet
+```
+
+Example return:
+```
+                           TxHash                                 TxIx        Amount
+--------------------------------------------------------------------------------------
+94461e17271b4a108f679eb7b6947aea29573296a5edca635d583fb40785e05d     0        1000000000 lovelace + TxOutDatumHashNone
+```
+
+### Drafting the transaction
+```sh
+cardano-cli transaction build-raw \
+--mary-era \
+--tx-in "94461e17271b4a108f679eb7b6947aea29573296a5edca635d583fb40785e05d#0" \
+--tx-in-script-file payment.script \
+--tx-out "$(cat payment.addr)"+0 \
+--fee 0
+--out-file tx.draft
+```
+
+### Calculating the fee
+*Note: if you don't have a `protocol.json` file available you obtain it with `cardano-cli`:*
+```sh
+cardano-cli query protocol-parameters \
+--out-file protocol.json \
+--testnet-magic 1097911063
+```
+
+Then we can calculate the fee:
+```sh
+cardano-cli transaction calculate-min-fee \
+--tx-body-file tx.draft  \
+--tx-in-count 1 \
+--tx-out-count 1 \
+--witness-count 2 \
+--mainnet \
+--protocol-params-file ./protocol.json
+```
+Example output:
+```
+194654 Lovelace
+```
+Note that an ordinary transaction looking like this would most likely require one signature. However, our payment part is defined with a script that requires two signatures. Therefore, we need two witnesses for this transaction.
+
+### Creating the unsigned transaction
+Now we can calculate the output amount (input minus the fee) and create the unsigned transaction.
+```sh
+expr 1000000000 - 194654
+```
+```sh
+cardano-cli transaction build-raw \
+--mary-era \
+--tx-in "94461e17271b4a108f679eb7b6947aea29573296a5edca635d583fb40785e05d#0" \
+--tx-in-script-file payment.script \
+--tx-out "$(cat payment.addr)"+999805346 \
+--fee 194654
+--out-file tx.raw
+```
+
+### Signing the transaction
+Because we need two witnesses, one from the key from hardware wallet and one from the key from `cardano-cli` we will create them separately and afterwards assemble them together to create a signed transaction.
```

Comment:
```suggestion
Because we need two witnesses, one from the key from hardware wallet and one from the key from `cardano-cli`, we will create them separately and afterwards assemble them together to create a signed transaction.
```